<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ozonetel-santosh" failures="3" tests="25" name="com.CA.qa.TestCases.ValidateConfigurationsPage" time="47.063" errors="0" timestamp="2021-01-19T18:24:03 IST" skipped="0">
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateLocationNameEmptyErrorMsg6_1" time="2.178"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateLocationRangeErrorMsg6_11" time="2.060"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateLocationRangeErrorMsg6_12" time="1.971"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateLocationRangeErrorMsg6_13" time="2.082"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingNameEmptyErrorMsg7_1" time="2.069"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingNameRangeErrorMsg7_11" time="2.002"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingNameRangeErrorMsg7_12" time="2.192"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingNameRangeErrorMsg7_13" time="2.057"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingDomainNameEmptyErrorMsg7_21" time="1.934"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingDomainNameRangeErrorMsg7_22" time="2.079"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingDomainNameRangeErrorMsg7_23" time="2.050"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingDomainNameRangeErrorMsg7_24" time="2.012"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingLocalIpEmptyErrorMsg7_3" time="1.980"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingLocalIpRangeErrorMsg7_31" time="2.067">
    <failure message="expected [Domain Name should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and Name cannot start and end with Special characters.] but found [Local IP should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and IP cannot start and end with Special characters.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Domain Name should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and Name cannot start and end with Special characters.] but found [Local IP should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and IP cannot start and end with Special characters.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.CA.qa.TestCases.ValidateConfigurationsPage.ValidateURLMappingLocalIpRangeErrorMsg7_31(ValidateConfigurationsPage.java:1002)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ValidateURLMappingLocalIpRangeErrorMsg7_31 -->
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingLocalIpRangeErrorMsg7_32" time="2.046">
    <failure message="expected [Domain Name should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and Name cannot start and end with Special characters.] but found [Local IP should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and IP cannot start and end with Special characters.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Domain Name should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and Name cannot start and end with Special characters.] but found [Local IP should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and IP cannot start and end with Special characters.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.CA.qa.TestCases.ValidateConfigurationsPage.ValidateURLMappingLocalIpRangeErrorMsg7_32(ValidateConfigurationsPage.java:1012)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ValidateURLMappingLocalIpRangeErrorMsg7_32 -->
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateURLMappingLocalIpRangeErrorMsg7_33" time="2.099">
    <failure message="expected [Domain Name should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and Name cannot start and end with Special characters.] but found [Local IP should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and IP cannot start and end with Special characters.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Domain Name should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and Name cannot start and end with Special characters.] but found [Local IP should be between 2 to 50 alphanumeric characters long allows special characters like .,@,_,space,- and IP cannot start and end with Special characters.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.CA.qa.TestCases.ValidateConfigurationsPage.ValidateURLMappingLocalIpRangeErrorMsg7_33(ValidateConfigurationsPage.java:1022)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ValidateURLMappingLocalIpRangeErrorMsg7_33 -->
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateSmsTemplateNameEmptyErrorMsg8_1" time="1.955"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateSmsTemplateNameRangeErrorMsg8_11" time="1.946"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateSmsTemplateNameRangeErrorMsg8_12" time="2.070"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateSmsTemplateNameRangeErrorMsg8_13" time="2.000"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidateSmsTemplateTextErrorMsg8_2" time="1.912"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidatedispositionEmptyErrorMsg9_1" time="1.053"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidatedispositionRangeErrorMsg9_11" time="1.134"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidatedispositionSplRangeErrorMsg9_12" time="1.031"/>
  <system-out/>
  <testcase classname="com.CA.qa.TestCases.ValidateConfigurationsPage" name="ValidatedispositionSplRangeErrorMsg9_13" time="1.084"/>
  <system-out/>
</testsuite> <!-- com.CA.qa.TestCases.ValidateConfigurationsPage -->
